gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -25,6 +25,11 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
         assert extension.getSeed() == null
     }
 
+    void testDefaultCategories() {
+        def extension = createExtension()
+        assert extension.getCategories() == AndroidCommandPluginExtension.CATEGORIES_DEFAULT
```

Comment:
We still use CATEGORIES_DEFAULT value when it's not specified and I think it brings more consistency across other parameters.